global_buffer/Makefile

```diff
@@ -57,16 +57,25 @@ SIMV = ./simv \
 	   -l vcs.log
 
 
-.PHONY: rdl clean compile html
+.PHONY: rdl_param rdl_final rdl_pre clean compile html param_files
 
-rdl: systemRDL/rdl_models/glb.rdl systemRDL/ordt_params/glb.parms
-	../systemRDL/perlpp.pl systemRDL/rdl_models/glb.rdl -o systemRDL/rdl_models/glb.rdl.final
+param_files: global_buffer_main.py
```

Comment:
eventually we should see if we can try to move some of this stuff into python itself

Reply:
good idea. After testbench and PD flow is done, let's see whether we can move this into python

Reply:
Good catch. I will update it to generate default global_buffer_param.